framework/src/testing/fixtures/accounts.ts

```diff
@@ -13,136 +13,66 @@
-	'a6f6a0543ae470c6b056021cb2ac153368eafeec',
-	'a9c66694dd65b2fdf40cdf45a0c308cbd38004fc',
-	'ab0041a7d3f7b2c290b5b834d46bdc7b7eb85815',
-	'abd2ed5ad35b3a0870aadae6dceacc988ba63895',
-	'acfbdbaeb93d587170c7cd9c0b5ffdeb7ff9daec',
-	'ad42f8e867d618171bf4982e64269442148f6e11',
-	'aebd99f07218109162a905d0e0c91e58bedc83c5',
-	'b11c5811ea074a30142d824b6e8cfd3df14b2688',
-	'b485becd88db1ab3d556d405204451ba00adaa7d',
-	'b543e2e592200beb38235f6e48f8abe1d87ad872',
-	'b56c55b9a70c8e2f07979b862374aed0e92a6dda',
-	'b7580969dd56151f608931f126f793bbf45d8fa0',
-	'b76a0f1819c4be0a1482567ca9b9fbed3eda444c',
-	'bd175729d4177259c71cf13fd4ecfb5d01542706',
-	'be89f4e983dfb04e2b58a12eb9ed18149e108b07',
-	'c3ab2ac23512d9bf62b02775e22cf80df814eb1b',
-	'c697b620c7c4015e32dd7bdd7d0430b33404e107',
-	'c98554123062ac5795a3ee905b081e863db5a818',
-	'ca309a5f4bbf11ca86592febb6d2ccc78309f69e',
-	'ca5f6d76eab6e4f5aacee2864c79034d7111b986',
-	'cb579ee537b34926d47129a0b54c0e6d00ef3004',
-	'd06fe6d3e5f7facb5855eca839422fe3824a5d6e',
-	'd0a0e45b950e3871d8783b973409042b4ab382d4',
-	'd2c9a93755aed20c4d8f55c1e92b812d2c7d49d2',
-	'd3c8064d011ef853e3be506b95a045f41f78e72a',
-	'd5bd2050b74b309d54819ca17add173c6fca1e16',
-	'd5c4e380b1ec2f7f2068cfba9a90cb3ae7816110',
-	'd5e1f52cbe4a11a3730b98f52109b57602a9c4a1',
-	'd8e611bafd70a549f035cf61ab0d6ed9e7f25c4e',
-	'dcb5bf35b6d521195e613c42483f520139e2331d',
-	'df0e187bb3895806261c87cf66e1772566ee8e58',
-	'e2950a9f07b44e724df2129360cc140293c08308',
-	'e39316cc020089ea7a5614bcf69a8931c10630a7',
-	'e9355152c117c9e1fad8be86e9abea961cef4a36',
-	'f730cb929a1c45032387c345e10d2427bea55a5e',
-	'fa526a1611ccc66dec815cb963174118074b736e',
-	'ffce8ce225c5d80098f50e877125b655aef6d101',
-];
+import { ModuleClass, PartialAccount } from '../types';
+import { getAccountSchemaFromModules } from '../utils';
+import { defaultConfig } from './config';
 
-export const defaultAccount = {
+const defaultPassword = 'elephant tree paris dragon chair galaxy';
+export const defaultFaucetAccount = {
 	address: Buffer.from('d04699e57c4a3846c988f3c15306796f8eae5c1c', 'hex'),
 	publicKey: Buffer.from('0fe9a3f1a21b5530f27f87a414b549e79a940bf24fdf2b2f05e7f22aeeecc86a', 'hex'),
 	passphrase: 'peanut hundred pen hawk invite exclude brain chunk gadget wait wrong ready',
 	balance: '10000000000000000',
 	encryptedPassphrase:
 		'iterations=10&cipherText=6541c04d7a46eacd666c07fbf030fef32c5db324466e3422e59818317ac5d15cfffb80c5f1e2589eaa6da4f8d611a94cba92eee86722fc0a4015a37cff43a5a699601121fbfec11ea022&iv=141edfe6da3a9917a42004be&salt=f523bba8316c45246c6ffa848b806188&tag=4ffb5c753d4a1dc96364c4a54865521a&version=1',
-	password: 'elephant tree paris dragon chair galaxy',
+	password: defaultPassword,
+};
+
+export const defaultAccounts = <T>(): PartialAccount<T>[] =>
+	defaultConfig.forging.delegates.map(
+		account => ({ address: Buffer.from(account.address, 'hex') } as PartialAccount<T>),
+	);
+
+const getDelegateFromDefaultConfig = (address: Buffer) => {
```

Comment:
I am not sure if this function should be in `fixtures`. maybe move it to `utils`.
Same for other functions below.